test/e2e/simple-deployment/chainsaw-test.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: simple-deployment
+spec:
+  description: | 
+    SimpleDeployment tests a ResourceGraphDefinition that creates a single Deployment
+    The test verifies
+    1. Creating RGD installs the Instance SimpleDeployment CRD
+    2. Creating SimpleDeployment instance creates a Deployment
+    3. Changing the replicas in the SimpleDeployment instance changes the Deployment
+    4. Deleting the SimpleDeployment instance deletes the Deployment
```

Comment:
I can't see where we delete the `SimpleDeployment` and check if the instance gets deleted.

You can add the following:
```
  - name: delete the rgd
    try:
    - delete:
        ref:
          apiVersion: kro.run/v1alpha1
          kind: ResourceGraphDefinition
          name: simple-deployment-rgd
  - name: check that the deployment is deleted
     try:
      - error:
          file: deployment-error.yaml
```

The file `deployment-error` should be the manifest that you expect to be deleted.